adam_core/orbits/ephemeris.py

```diff
@@ -0,0 +1,109 @@
+import pandas as pd
+from quivr import Float64Column, StringAttribute, StringColumn, Table
+from typing_extensions import Self
+
+from ..coordinates.spherical import SphericalCoordinates
+from ..observers import Observers
+
+
+class Ephemeris(Table):
+
+    orbit_id = StringColumn(nullable=False)
+    object_id = StringColumn()
+    observer = Observers.as_column(nullable=False)
+    rho = Float64Column()
+    ra = Float64Column(nullable=False)
+    dec = Float64Column(nullable=False)
+    vrho = Float64Column()
+    vra = Float64Column()
+    vdec = Float64Column()
+    frame = StringAttribute()
```

Comment:
I'm a little surprised this doesn't embed SphericalCoordinates directly as a `SphericalCoordinates.as_column()`. Why not? It would be nice to bring covariances in, and that seems like an easy way.

Reply:
I'd actually prefer it to be `SphericalCoordinates.as_column()` but what I was thinking was that it makes sense to have a bit of symmetry with our future `Observations` table and `Ephemeris` since we directly compare those for residual calculations. For `Observations` we discussed not having SphericalCoordinates included since for example rho, vrho, vra, vdec will be empty for the vast majority of our observations. Although that said, I guess `Residuals.calculate(Observations.to_spherical_coordinates(), Ephemeris.coordinates)` really isn't that cumbersome?

Reply:
One other thought too is that times would be duplicated between Observers and SphericalCoordinates. Not sure how to handle that. I think the duplication is okay for now but it would be cool to be able to "link" between tables.